app/Http/Controllers/FoodController.php

```diff
@@ -3,8 +3,73 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Food;
+use App\Category;
+use App\Http\Requests\FoodPostRequest;
+use Image;
+use Session;
 
 class FoodController extends Controller
 {
-    //
+    protected $food;
+    
+    /**
+     * FoodController constructor.
+     *
+     * @param Food $food dependence injection
+     */
+    public function __construct(Food $food)
+    {
+        $this->food = $food;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
```

Comment:
Please remove this function. 
**Admin-Create-Food**. This pull request should be contain functions about Create Food feature